dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/startup/TableAlteror.java

```diff
@@ -1092,6 +1092,14 @@ public void execute()
         executeSql( "alter table userroledataset drop constraint fk_userroledataset_datasetid;" );
         executeSql( "alter table userroledataset drop constraint fk_userroledataset_userroleid;" );
         
+        // Update leaderOnlyJob flag in jobconfiguration

```

Comment:
Since the new attribute is "not-null" should you set all other rows to false, or given a certain JobType to true?

Reply:
Discussed with Stian, all the existing jobs (apart from the newly added leader election job), leaderOnlyJob will be true. Do you think that list is not exhaustive? Maybe we can change the scripts to set everything as true with 
```
update jobconfiguration set leaderonlyjob=true
```

and then explicitly set leaderelectionjob to false. Sounds ok?

Reply:
Ok, I thought maybe some of the jobTypes were configurable. In that case users may have added other jobs which also would have to be set by this script.